sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -1216,6 +1216,7 @@ public void listRevisionsWithPaginationAndRepeatIterator(HttpClient httpClient,
      * Verifies that, given a ton of existing settings, we can list the ConfigurationSettings using pagination
      * (ie. where 'nextLink' has a URL pointing to the next page of results.
      */
+    @Disabled("Error code 403 TOO_MANY_REQUESTS https://github.com/Azure/azure-sdk-for-java/issues/36602")
```

Comment:
I am not sure whether we need to change the SDK code in this branch, for the Spring Boot 3 branch, we sync the SDK code from the main branch, and take it as is. We only change the Spring-related code there, use the same version of SDK as the main branch.

Reply:
The issue is that the code in main is a beta release and we ran into build issues with it as there was an issue with it compared to the released version. The work around was to temp undo the change in this pr, which should be fine as it is going into a feature branch.